src/Confluent.Kafka/Serializers.cs

```diff
@@ -0,0 +1,196 @@
+    /// </summary>
+    public static class Serializers
+    {
+        /// <summary>
+        ///     Encodes a string value in a byte array.
+        /// </summary>
+        public static Serializer<string> UTF8 = (topic, data) =>
+        {
+            if (data == null)
+            {
+                return null;
+            }
+
+            return Encoding.UTF8.GetBytes(data);
+        };
+
+        /// <summary>
+        ///     Encodes a Null value to null.
+        /// </summary>
+        public static Serializer<Null> Null = (topic, data) => null;
+
+        /// <summary>
+        ///     Serializes the specified <see cref="System.Int64"/> value to a byte array of length 8. Byte order is big endian (network byte order).
+        /// </summary>
+        /// <param name="data">
+        ///     The <see cref="System.Int64"/> value to serialize.
+        /// </param>
+        /// <param name="topic">
+        ///     The topic associated with the data (ignored by this serializer).
+        /// </param>
+        /// <returns>
+        ///     The <see cref="System.Int64"/> value <paramref name="data" /> encoded as a byte array of length 8 (network byte order).
+        /// </returns>
+        public static byte[] Long(string topic, long data)
+        {
+            var result = new byte[8];
+            result[0] = (byte)(data >> 56);
+            result[1] = (byte)(data >> 48);
+            result[2] = (byte)(data >> 40);
+            result[3] = (byte)(data >> 32);
+            result[4] = (byte)(data >> 24);
+            result[5] = (byte)(data >> 16);
+            result[6] = (byte)(data >> 8);
+            result[7] = (byte)data;
+            return result;
+        }
+
+        /// <summary>
+        ///     Serializes the specified <see cref="System.Int32"/> value to a byte array of length 4. Byte order is big endian (network byte order).
+        /// </summary>
+        /// <param name="data">
+        ///     The <see cref="System.Int32"/> value to serialize.
+        /// </param>
+        /// <param name="topic">
+        ///     The topic associated with the data (ignored by this serializer).
+        /// </param>
+        /// <returns>
+        ///     The <see cref="System.Int32"/> value <paramref name="data" /> encoded as a byte array of length 4 (network byte order).
+        /// </returns>
+        public static byte[] Int32(string topic, int data)
+        {
+            var result = new byte[4]; // int is always 32 bits on .NET.
+            // network byte order -> big endian -> most significant byte in the smallest address.
+            // Note: At the IL level, the conv.u1 operator is used to cast int to byte which truncates
+            // the high order bits if overflow occurs.
+            // https://msdn.microsoft.com/en-us/library/system.reflection.emit.opcodes.conv_u1.aspx
+            result[0] = (byte)(data >> 24);
```

Comment:
there are builtin methods for int swapping:
https://docs.microsoft.com/en-us/dotnet/api/system.net.ipaddress.hosttonetworkorder?view=netframework-4.7.2

Reply:
added to #614. I probably did this as a perf optimization, but it's probably not significant. reflects previous code.